app/src/main/res/values/dimens.xml

```diff
@@ -137,4 +138,8 @@
     <dimen name="height_minimum">2dp</dimen>
     <dimen name="height_room_recycler">20dp</dimen>
     <dimen name="width_room_recycler">20dp</dimen>
+    <dimen name="room_image_weight">1</dimen>
+    <dimen name="room_name_weight">4</dimen>
+    <dimen name="room_delete_weight">1</dimen>
```

Comment:
No need to declare weight here, these are not dimensions.

Reply:
Updated. Please review now